library/lib/core/application.cpp

```diff
@@ -136,23 +136,70 @@ void Application::createWindow(std::string windowTitle)
     // Load fonts and setup fallbacks
     Application::platform->getFontLoader()->loadFonts();
 
+    NVGcontext* vg = Application::getNVGContext();
+    std::string locale = Application::getLocale();
     int regular = Application::getFont(FONT_REGULAR);
-    if (regular != FONT_INVALID)
+
+    // when defined font stash exists, determine regular font and try to build fallback chain, all fallback failures won't logged
+    int standard = Application::getFont(FONT_STANDARD_REGULAR);
+    int schinese = Application::getFont(FONT_SCHINESE_REGULAR);
+    // int extschinese = Application::getFont(FONT_SCHINESE_EXTEND);
+    int tchinese = Application::getFont(FONT_TCHINESE_REGULAR);
+    int korean = Application::getFont(FONT_KOREAN_REGULAR);
+    std::string regularRedir = "nope";
+    if (regular == FONT_INVALID)
     {
-        NVGcontext* vg = Application::getNVGContext();
+        Logger::info("no regular font loaded at the moment, checking the rest of font stash");
```

Comment:
Log messages should start with uppercase letters :p

Reply:
Also there's no need to say "at the moment" since it's not expected to change at any later moment